### PR TITLE
Specify TRNG timing requirement

### PR DESCRIPTION
**Pull request by andreslagarcavilla**
Address #136

### CODE REVIEW

**Comment by andreslagarcavilla** [comment]
@korran is OOO, assigning to a few other reviewers. Thanks

**Review by FerralCoder — APPROVED** [review]

File: docs/CaliptraIntegrationSpecification.md [hunk 1]
```diff
@@ -476,6 +476,10 @@ Having an interface that is separate from the SoC mailbox ensures that this requ
 
 TRNG DATA register is tied to TRNG VALID PAUSER. SoC can program the TRNG VALID PAUSER and lock the register using TRNG\_PAUSER\_LOCK[LOCK]. This ensures that TRNG DATA register is read-writeable by only the PAUSER programmed into the TRNG\_VALID\_PAUSER register. If the CPTRA\_TNRG\_PAUSER\_LOCK.LOCK is set to ‘0, then any agent can write to the TRNG DATA register. If the lock is set, only an agent with a specific TRNG\_VALID\_PAUSER can write.
 
+The ROM and firmware currently time out on the TRNG interface after 250,000
+attempts to read a DONE bit set in the architectural registers, as alluded to in
```
Comment (steph-morton) [inline docs/CaliptraIntegrationSpecification.md:480]: recommend update to: `attempts to read a DONE bit. This bit is set in the architectural registers, as referenced in`

File: docs/CaliptraIntegrationSpecification.md [hunk 2]
```diff
@@ -476,6 +476,10 @@ Having an interface that is separate from the SoC mailbox ensures that this requ
...
+3. above.
```
Comment (steph-morton) [inline docs/CaliptraIntegrationSpecification.md:481]: recommend update to `3, in the preceding list.`

**Review by calebofearth — DISMISSED** [review]
Looks good to me, after Steph's edits.

File: docs/CaliptraIntegrationSpecification.md (hunk 2 above)
Comment (andreslagarcavilla) [reply docs/CaliptraIntegrationSpecification.md]: Applied

File: docs/CaliptraIntegrationSpecification.md (hunk 1 above)
Comment (andreslagarcavilla) [reply docs/CaliptraIntegrationSpecification.md]: Applied

**Comment by andreslagarcavilla** [comment]
Basically @steph-morton rewrote the PR :) -- changes applied

File: docs/CaliptraIntegrationSpecification.md [hunk 3]
```diff
@@ -477,8 +477,8 @@ Having an interface that is separate from the SoC mailbox ensures that this requ
 TRNG DATA register is tied to TRNG VALID PAUSER. SoC can program the TRNG VALID PAUSER and lock the register using TRNG\_PAUSER\_LOCK[LOCK]. This ensures that TRNG DATA register is read-writeable by only the PAUSER programmed into the TRNG\_VALID\_PAUSER register. If the CPTRA\_TNRG\_PAUSER\_LOCK.LOCK is set to ‘0, then any agent can write to the TRNG DATA register. If the lock is set, only an agent with a specific TRNG\_VALID\_PAUSER can write.
 
 The ROM and firmware currently time out on the TRNG interface after 250,000
-attempts to read a DONE bit set in the architectural registers, as alluded to in
-3. above.
+attempts to read a DONE bit. This bit is set in the architectural registers, as
+referenced in 3. in the preceding list.
```
Comment (steph-morton) [inline docs/CaliptraIntegrationSpecification.md:481]: I think you can also delete the `.` after 3.

**Review by steph-morton — APPROVED** [review]

File: docs/CaliptraIntegrationSpecification.md (hunk 3 above)
Comment (andreslagarcavilla) [reply docs/CaliptraIntegrationSpecification.md]: Done

**Review by steph-morton — APPROVED** [review]

**Review by mcockrell-google — APPROVED** [review]

**Pull request merged**